astropy/io/fits/diff.py

```diff
@@ -290,7 +305,17 @@ def __init__(self, a, b, ignore_keywords=[], ignore_comments=[],
         self.ignore_fields = set(k.upper() for k in ignore_fields)
 
         self.numdiffs = numdiffs
-        self.tolerance = tolerance
+        self.rtol = rtol
+        self.atol = atol
+
+        if tolerance is not None:
```

Comment:
Maybe add a in-code comment that this can be removed for v3.1 and after? Is that numbering right, @mhvk? Sorry, I am still learning the proper deprecated code removal policy.